src/containers/Main/SiteLayout.tsx

```diff
@@ -19,88 +19,108 @@ import {
-        </Route>
-        <Route exact path="/notifications/:notificationId">
-          <Notification />
-        </Route>
-        <Route exact path="/products">
-          <ProductList />
-        </Route>
-        <Route exact path="/products/:productId">
-          <Product />
-        </Route>
-        <Route>
-          <NoMatch />
-        </Route>
-      </Switch>
+      <Container>
+        <UserArea />
+        <DevOptions />
+      </Container>
+      <Container className="basebody" style={containerStyle}>
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route exact path="/login">
+            <Login />
+          </Route>
+          <Route exact path="/applications">
+            <ListWrapper />
+          </Route>
+          <Route path="/application/new">
+            <ApplicationProvider>
+              <ApplicationCreate />
+            </ApplicationProvider>
+          </Route>
+          <Route path="/application/:serialNumber">
+            <FormElementUpdateTrackerProvider>
+              <ApplicationWrapper />
+            </FormElementUpdateTrackerProvider>
+          </Route>
+          <Route exact path="/application/:serialNumber/approval">
+            <Approval />
+          </Route>
+          <Route exact path="/admin">
+            <Admin />
+          </Route>
+          <Route exact path="/admin/templates">
+            <TemplateList />
+          </Route>
+          <Route exact path="/admin/templates/new">
+            <TemplateNew />
+          </Route>
+          <Route exact path="/admin/templates/:templateId/:step">
+            <Template />
+          </Route>
+          <Route exact path="/admin/users">
+            <AdminUsers />
+          </Route>
+          <Route exact path="/admin/permissions">
+            <AdminPermissions />
+          </Route>
+          <Route exact path="/admin/config">
+            <Config />
+          </Route>
+          <Route exact path="/account">
+            <Account />
+          </Route>
+          <Route exact path="/organisations/:orgName">
+            <Organisation />
+          </Route>
+          <Route exact path="/organisations/:orgName/members">
+            <OrgMemberEdit />
+          </Route>
+          <Route exact path="/notifications">
+            <NotificationsList />
+          </Route>
+          <Route exact path="/notifications/:notificationId">
+            <Notification />
+          </Route>
+          <Route exact path="/products">
+            <ProductList />
+          </Route>
+          <Route exact path="/products/:productId">
+            <Product />
+          </Route>
+          <Route>
+            <NoMatch />
+          </Route>
+        </Switch>
+      </Container>
+      <Container>
+        <Footer />
+      </Container>
     </Router>
   )
 }
 
+// Styles - TODO: Move to LESS || Global class style (semantic)
+const containerStyle = {
+  minHeight: '100vh',
```

Comment:
Not sure why we need this. Once you turn the Footer off "fixed" (my other suggestion), it disappears because of this. Maybe we don't need it at all, but if we want it, then I wouldn't go more than 90vh.

Reply:
Done: e86b53a